src/main/java/uk/gov/hmcts/reform/hmc/data/LinkedGroupDetails.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.hmc.data;
+
+import lombok.Data;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+
+@Table(name = "linked_group_details")
+@Entity
+@Data
+
+public class LinkedGroupDetails {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY,
+        generator = "linked_group_details_id_seq")
+    @Column(name = "linked_group_id")
+    private Long linkedGroupId;
```

Comment:
do we need linked prefix to the fields, as the class name is already signifying what it is doing